packages/eslint-plugin-pf-codemods/src/rules/helpers/hasCodemodDataTag.ts

```diff
@@ -0,0 +1,13 @@
+import { JSXAttribute, JSXOpeningElement } from "estree-jsx";
+import { getAttributeName } from "./getAttributeName";
+
+/** Returns true if the passed opening element has a data-codemods attribute */
+export function hasCodeModDataTag(openingElement: JSXOpeningElement) {
```

Comment:
Could always be a separate helper, but just wondering if we should only check if the data tag exists by name, or return the actual attribute and base conditional logic on that. Just thinking if we ever start supplying specific values to a `data-codemods` attribute, e.g. maybe we only want to skip/run a rule if an openingElement has `data-codemods="renamed"`, but not `data-codemods="deprecated"` or something.